src/foam/nanos/notification/EmailSetting.js

```diff
@@ -91,18 +93,19 @@ foam.CLASS({
           if ( config != null ) {
             notification.getEmailArgs().put("link", config.getUrl());
           }
-        }    
+        }
+
+        if ( "NOC".equals(notification.getEmailName()) && ! SafetyUtil.isEmpty(notification.getBody()) ) {
```

Comment:
NOC notification doesnt set an email args for now and some notifications not storing its body after notification translation stuffs updated. So temporarily added the condition if "NOC".equals(notification.getEmailName()) but in the near future when going through this
https://nanopay.atlassian.net/browse/NP-3353, might be considered here.

Reply:
I don't care, simply remove all of this 'NOC' testing on the foam side. 
It's not the foam side's responsibility.